packages/utils/src/lib/file-system.ts

```diff
@@ -61,11 +62,10 @@ export function logMultipleFileResults(
   const succeededCallback = (result: PromiseFulfilledResult<FileResult>) => {
     const [fileName, size] = result.value;
     const formattedSize = size ? ` (${chalk.gray(formatBytes(size))})` : '';
-    console.info(`- ${chalk.bold(fileName)}${formattedSize}`);
-  };
-  const failedCallback = (result: PromiseRejectedResult) => {
-    console.warn(`- ${chalk.bold(result.reason)}`);
+    return `- ${chalk.bold(fileName)}${formattedSize}`;
   };
+  const failedCallback = (result: PromiseRejectedResult) =>
+    `- ${chalk.bold(result.reason)}`;
 
   logMultipleResults<FileResult>(
```

Comment:
Note: The `logMultipleResults` deserves a refactor. This code seems too complex for what it actually does.

Reply:
could you open an issue pls

Reply:
#557